README.md

```diff
@@ -58,30 +58,29 @@ Next, step through the following commands to setup secure and private communicat
 # This will create a Space and Project for you in Ockam Orchestrator and provision an
 # End-to-End Encrypted Cloud Relay service in your `default` project at `/project/default`.
 ockam enroll
-ockam project information --output json > default-project.json
 
 # -- APPLICATION SERVICE --
 
-# Start an application service, listening on a local ip and port, that clients would access
-# through the cloud encrypted relay. We'll use a simple http server for this first example
+# Start an application service, listening on a local IP and port, that clients would access
+# through the cloud encrypted relay. We'll use a simple HTTP server for this first example
 # but this could be any other application service.
 python3 -m http.server --bind 127.0.0.1 5000
 
-# In a new terminal window, setup an ockam node, called `s`, as a sidecar next to the 
-# application service. Then create a tcp outlet, on the `s` node, to send raw tcp traffic
-# to the service. Finally create a relay in your default Orchestrator project.
-ockam node create s --project-path default-project.json
-ockam tcp-outlet create --at s --from /service/outlet --to 127.0.0.1:5000
-ockam relay create s --at /project/default --to s
+# In a new terminal window, setup a tcp-outlet that makes a TCP service available at the given
+# address `5000`. We can use this to send raw TCP traffic to the HTTP server on port `5000`.
+# Finally create a relay in your default Orchestrator project. Relays make it possible to
+# establish end-to-end protocols with services operating in a remote private networks, without
+# requiring a remote service to expose listening ports to an outside hostile network like the
+# Internet.
+ockam tcp-outlet create --to 5000
+ockam relay create
 
 # -- APPLICATION CLIENT --
 
-# Setup an ockam node, called `c`, as a sidecar next to our application client. Then create
-# an end-to-end encrypted secure channel with s, through the cloud relay. Finally, tunnel
-# traffic from a local tcp inlet through this end-to-end secure channel.
-ockam node create c --project-path default-project.json
-ockam secure-channel create --from c --to /project/default/service/forward_to_s/service/api\
-  | ockam tcp-inlet create --at c --from 127.0.0.1:7000 --to -/service/outlet
+# Setup a a local tcp inlet to allow raw TCP traffic to be received on port `7000` before
```

Comment:
I guess it should either be `tcp-inlet` or `TCP inlet` here

Reply:
Done 👍